lib/Translation/ImportDataExtractor/Xliff12DataExtractor.php

```diff
@@ -83,6 +83,10 @@ public function extractElement(string $importId, int $stepId): ?AttributeSet
             $content = $transUnit->target->asXml();
             $content = $this->xliffEscaper->unescapeXliff($content);
 
+            if (is_string($content)) {
+                $content = html_entity_decode(trim($content));
```

Comment:
I think here would be a better place for doing this: 
https://github.com/pimcore/pimcore/blob/514ffeb569f6305ce2e18009014ffeb8432c0bbc/lib/Translation/Escaper/Xliff12Escaper.php#L101
Actually it is already decoded, but only when wrapped in <bpt> tags. 
Maybe we can just add an else clause there, where we do the decode in case it's just a simple string and no markup. WDYT?

Reply:
@valentin- 🏓😉

Reply:
@brusch sorry I was quite busy and then oof again